composer.json

```diff
@@ -16,7 +16,7 @@
     ],
     "require": {
         "composer/installers": "^1.9",
-        "drupal/admin_toolbar": "^3.1",
+        "drupal/admin_toolbar": "^3.2.1",
```

Comment:
note the module change here, appears to be the admin_toolbar module as per this pull request: https://github.com/huzainahshah1/drupal9_sitebuilding/pull/4